recipes-extended/netconfig/netconfig.bb

```diff
@@ -14,7 +14,7 @@ do_install_append() {
     install -m 0644 ${WORKDIR}/*.network ${D}${sysconfdir}/systemd/network
 }
 
-pkg_postinst_ontarget_${PN} () {
+pkg_postinst_${PN} () {
```

Comment:
Could you please share the background of this change? I suppose this wouldn't be working as expected.